cloud/pkg/devicecontroller/utils/crdclient.go

```diff
@@ -24,7 +24,7 @@ func NewCRDClient(cfg *rest.Config) (*rest.RESTClient, error) {
 	config.APIPath = "/apis"
 	config.GroupVersion = &v1alpha1.SchemeGroupVersion
 	config.ContentType = runtime.ContentTypeJSON
-	config.NegotiatedSerializer = serializer.DirectCodecFactory{CodecFactory: serializer.NewCodecFactory(scheme)}
+	config.NegotiatedSerializer = serializer.NewCodecFactory(scheme).WithoutConversion()
```

Comment:
change needed from k8s 1.17? what's will happen without this change?  could you pls say something in the commit message? thanks!

Reply:
Yes, there's some refactor in k8s.io/apimachinery, without this change, the crd client can't talk to kube-apiserver.